compiler/plugins/target/ROCM/ROCMTarget.cpp

```diff
@@ -76,6 +81,34 @@ struct ROCmOptions {
                  "specific unprefixed microkernels to enable, e.g. `mmt4d`."));
   }
 
+  LogicalResult verify(mlir::Builder &builder) const {
+    if (GPU::normalizeHIPTarget(targetChip).empty()) {
+      return emitError(builder.getUnknownLoc(), "Unknown ROCm target '")
+             << targetChip << "'";
+    }
+    SmallVector<StringRef> features;
+    llvm::SplitString(targetFeatures, features, ",");
+    for (StringRef f : features) {
+      if (f.ends_with("+") || f.ends_with("-") ||
+          !(f.starts_with("+") || f.starts_with("-"))) {
```

Comment:
This seems redundant? I think we only need to check that it doesn't start with either `+` or `-`